NuimoSDK/NuimoLedMatrix.cs

```diff
@@ -5,19 +5,23 @@ namespace NuimoSDK
 {
     public class NuimoLedMatrix
     {
-        public bool[] Leds { get; }
-
         private const int LedCount = 81;
         private readonly char[] _ledOffCharacters = " 0".ToCharArray();
 
+        public NuimoLedMatrix()
+        {
+        }
+
```

Comment:
What's the reason behind having this empty constructor? The instantiated matrix will not be properly initialized, i.e. `Leds` will be empty.